src/services/twoActionsDialog/twoActionsDialogService.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Notifier } from '../../utils';
+import { SimpleTwoActionsDialog } from '@cosmotech/ui';
+
+const twoActionsDialogNotifier = new Notifier();
+
+class TwoActionsDialogClass {
+  openDialog(dialogProps) {
+    return new Promise((resolve) => {
+      dialogProps.resolve = resolve;
+      twoActionsDialogNotifier.send(dialogProps);
+    });
+  }
+}
+
+const TwoActionsDialogGlobal = () => {
+  const suscription = useRef();
```

Comment:
Typo: `suscription` should be `subscription`

Reply:
done